sunpy/io/special/asdf/entry_points.py

```diff
@@ -37,12 +37,22 @@ def get_extensions():
     from sunpy.io.special.asdf.converters.generic_map import GenericMapConverter
 
     sunpy_converters = [GenericMapConverter()] + SUNPY_FRAME_CONVERTERS
+    # order here is important as asdf will pick the first matching
+    # extension, start with the newest
+    _manifest_uris = [
+        #"asdf://sunpy.org/sunpy/manifests/sunpy-1.2.0",
+        #"asdf://sunpy.org/sunpy/manifests/sunpy-1.1.0",
```

Comment:
Have to avoid these otherwise the tests all fail.

Reply:
I didn't realize the 5.0 branch didn't include https://github.com/sunpy/sunpy/pull/7239 (which added the sunpy-1.1.0 manifest). At least the changes in that PR (and the 1.2.0 manifest uncommented here) will be required for this backport to enable ASDF standard 1.6.0 compatibility (since the 1.2.0 manifest added here includes references to the schemas added in that PR).

I'm not sure if there are other asdf related PRs that also will need to be backported for this to work.

Also, I see there's a 5.1 branch which "accepted" this backport with less issues:
https://github.com/sunpy/sunpy/commit/aa488a7345847ad9658763d96c2d999c9fd39cba

Is it required to also backport this to 5.0? If not I'd say given the issues above I'd say we close this PR.

Reply:
No there is no need to backport this to 5.0 if its too complex.

Yeah 5.1 took the backport with no problems. I didn't want to work out what else I needed to add but I can do so for 5.0 if you need.

Reply:
Thanks! I'm ok at this point to not have this backported to 5.0.

Reply:
In that case I will drop this.